### PR TITLE
2 customizables collector params

### PR DESCRIPTION
**Pull request by ccnifo**
Make _collector_query_workers_ and _collector_max_file_descriptors_ customizable. For the latest, avoid adding it to the conf if it's not set in hiera.

### CODE REVIEW

File: templates/22_manager.config.erb [hunk 1]
```diff
@@ -7,7 +7,12 @@ COLLECTOR_NAME = <%= @collector_name -%>
 START = FALSE
 
 ## Max number of processes to deal with query requests
-COLLECTOR_QUERY_WORKERS	= 16
+COLLECTOR_QUERY_WORKERS	= <%= @collector_query_workers -%>
+
+<% unless @collector_max_file_descriptors.nil? %>
```
Comment (olifre) [inline templates/22_manager.config.erb]: I think I'd prefer `-%>` here to not add to many empty lines to the file ;-).

File: templates/22_manager.config.erb [hunk 2]
```diff
@@ -7,7 +7,12 @@ COLLECTOR_NAME = <%= @collector_name -%>
...
+## Under Unix, maximum number of file descriptors to allow the collector to use.
+COLLECTOR_MAX_FILE_DESCRIPTORS = <%= @collector_max_file_descriptors -%>
+<% end %>
```
Comment (olifre) [inline templates/22_manager.config.erb]: Same here.

**Comment by ccnifo** [comment]
Both fixed, thanks!

**Comment by olifre** [comment]
Thanks!

**Pull request merged**